charts/argo-workflows/README.md

```diff
@@ -118,7 +118,7 @@ Fields to note:
 | crds.keep | bool | `true` | Keep CRDs on chart uninstall |
 | createAggregateRoles | bool | `true` | Create ClusterRoles that extend existing ClusterRoles to interact with Argo Workflows CRDs. |
 | emissary.images | list | `[]` | The command/args for each image on workflow, needed when the command is not specified and the emissary executor is used. |
-| extraObjects | list | `[]` | Array of extra K8s manifests to deploy |
+| extraObjects | list | `[]` | Array of extra kubernetes manifests to deploy |
```

Comment:
If you're going to long-form it, please give it an uppercase K as it's a proper noun.

Reply:
I think this change should be in its own PR. Making "K8s" into "Kubernetes" is not relevant to checking whether the server is enabled or not.

Reply:
You are right. I revert these changes and commit them in the next PR.